tests/integration-tests/tests/configure/test_pcluster_configure.py

```diff
@@ -161,3 +191,73 @@ def assert_config_contains_expected_values(
             validator.get("parameter_name")
         )
         assert_that(observed_value).is_equal_to(validator.get("expected_value"))
+
+
+def orchestrate_pcluster_configure_stages(
+    region,
+    key_name,
+    scheduler,
+    os,
+    instance,
+    vpc_id,
+    headnode_subnet_id,
+    compute_subnet_id,
+    vpc_stack,
+    omitted_subnets_num=0,
+):
+    compute_units = "vcpus" if scheduler == "awsbatch" else "instances"
+    # Due to the order of creation,PublicSubnet is the first in the list of subnets. Therefore it is the default value.
+    vpc_stack_public_subnet_id = vpc_stack.cfn_outputs["PublicSubnetId"]
+    # When there are omitted subnets, a note should be printed
+    omitted_note = "Note:  {0} subnet.+not listed.+".format(omitted_subnets_num) if omitted_subnets_num else ""
+    stage_list = [
+        {"prompt": r"AWS Region ID \[.*\]: ", "response": region},
+        {"prompt": r"EC2 Key Pair Name \[.*\]: ", "response": key_name},
+        {"prompt": r"Scheduler \[slurm\]: ", "response": scheduler},
+        {"prompt": r"Operating System \[alinux2\]: ", "response": os, "skip_for_batch": True},
+        {"prompt": fr"Minimum cluster size \({compute_units}\) \[0\]: ", "response": "1"},
+        {"prompt": fr"Maximum cluster size \({compute_units}\) \[10\]: ", "response": ""},
+        {"prompt": r"Master instance type \[t2\.micro\]: ", "response": instance},
+        {"prompt": r"Compute instance type \[t2\.micro\]: ", "response": instance, "skip_for_batch": True},
+        {"prompt": r"Automate VPC creation\? \(y/n\) \[n\]: ", "response": "n"},
+        {"prompt": r"VPC ID \[vpc-.+\]: ", "response": vpc_id},
+        {"prompt": r"Automate Subnet creation\? \(y/n\) \[y\]: ", "response": "n"},
+        {
+            "prompt": fr"{omitted_note}Master Subnet ID \[{vpc_stack_public_subnet_id}\]: ",
+            "response": headnode_subnet_id,
+        },
+        {
+            "prompt": fr"{omitted_note}Compute Subnet ID \[{vpc_stack_public_subnet_id}\]: ",
+            "response": compute_subnet_id,
+        },
+    ]
+    # When a user selects Batch as the scheduler, pcluster configure does not prompt for OS or compute instance type.
+    return [stage for stage in stage_list if scheduler != "awsbatch" or stage.get("skip_for_batch") is None]
```

Comment:
Can change `stage.get("skip_for_batch") is None` to `not stage.get("skip_for_batch")`, because `skip_for_batch` could be False
```
return [stage for stage in stage_list if scheduler != "awsbatch" or not stage.get("skip_for_batch")]
```